src/app/component/Header/Header.style.scss

```diff
@@ -81,6 +81,14 @@
             }
         }
 
+        &_cart_overlay {
+            .Header-Button_type_minicart {
+                @include after-mobile {
+                    --header-color: var(--header-desktop-button-color);
```

Comment:
Hmm, just make sure the `var(--header-desktop-button-color);` is right var to use. Prefer more low-level if possible, i.e. `--primary-base-color`.

Reply:
minicart icon now uses `--primary-base-color`